Formula/maggot.rb

```diff
@@ -0,0 +1,36 @@
+class Maggot < Formula
+  include Language::Python::Virtualenv
+
+  desc "GUI frontend for Msc-generator"
+  homepage "https://sourceforge.net/p/msc-generator"
+  url "https://downloads.sourceforge.net/project/msc-generator/maggot/v1.x/maggot-1.3.3-py3-none-any.whl"
+  sha256 "98fda1ec7db109d6845ebe795c75d54a9fcba76ced1ea9ccf44b85b0c85a9065"
+  license "GPL-3.0-or-later"
+
+  depends_on "msc-generator"
+  depends_on "python@3.9"
+
+  resource "pycairo" do
+    url "https://files.pythonhosted.org/packages/9d/6e/499d6a6db416eb3cdf0e57762a269908e4ab6638a75a90972afc34885b91/pycairo-1.20.0.tar.gz"
+    sha256 "5695a10cb7f9ae0d01f665b56602a845b0a8cb17e2123bfece10c2e58552468c"
+  end
+
+  resource "PyGObject" do
+    url "https://files.pythonhosted.org/packages/3a/a7/de282a4aaedba59d60a895a7821e6497b39cbdfa94a352776ff45ffc6e6f/PyGObject-3.38.0.tar.gz"
+    sha256 "051b950f509f2e9f125add96c1493bde987c527f7a0c15a1f7b69d6d1c3cd8e6"
+  end
+
+  def install
+    venv = virtualenv_create libexec, "python3"
+    venv.pip_install resources
+    # (pip|virtualenv)_install do not install .whl implicitly so we tell it to
+    venv.pip_install_and_link "#{buildpath}/maggot-1.3.3-py3-none-any.whl"
+  end
+
+  test do
+    # Try to open a file to have the machinery start up.
+    # It will exit with an error as the file is non-existent but we don't care
+    # as long as we see the stdout greeting.
+    assert_match "Maggot v#{version.major_minor}, welcome!", shell_output("maggot /no-such-file 2>/dev/null")
```

Comment:
```suggestion
    assert_match "Maggot v#{version.major_minor}, welcome!", shell_output("maggot /no-such-file 2>/dev/null", 1)
```
I think you need to add that you were expecting an exit code of `1`, or else the test assumes you encountered an error.